src/utils/changelog.ts

```diff
@@ -11,6 +11,7 @@ import { getVersion } from './version';
  */
 export const DEFAULT_CHANGELOG_PATH = 'CHANGELOG.md';
 export const DEFAULT_UNRELEASED_TITLE = 'Unreleased';
+export const SKIP_CHANGELOG_MAGIC_WORD = '#skip-changelog';
```

Comment:
Any considerations around escaping? In #296 we added escaping of #, I don't _think_ this should conflict but we are using the same OCTOTHORPE character here so something to think about.

Reply:
No need to escape as this does not end up in markdown.